trieste/acquisition/multi_objective/scalarization.py

```diff
@@ -0,0 +1,212 @@
+        self, models: Mapping[Tag, HasTrajectorySampler], datasets: Mapping[Tag, Dataset]
+    ) -> None:
+        """
+        Generate all the internal variables on initialization. For example, weights in a linear
+        weighted sum scalarization could be sampled.
+        """
+
+    @abstractmethod
+    def update(
+        self, models: Mapping[Tag, HasTrajectorySampler], datasets: Mapping[Tag, Dataset]
+    ) -> None:
+        """
+        Update all the internal variables. Meant to be used between the bayesian optimization
+        steps to change the state of the variables. For example, weights in a linear weighted
+        sum scalarization could be resampled.
+        """
+
+
+class Chebyshev(Scalarizer):
+    """
+    A popular scalarization function, improvement over linear weighted sum in that it can deal
+    with non-convex Pareto fronts. Along with weights, it has an ideal point parameter
+    which acts as a reference point for computing the Chebyshev distance.
+    In case we know the minimum of the Pareto front, we can set it to that. Otherwise
+    a function can be used that would adaptively choose the ideal point based on the models or
+    acquired data.
+
+    By setting ``alpha`` to a value larger than zero we can switch to augmented Chebyshev
+    scalarisation function which can help with avoiding weakly Pareto optimal solutions.
+
+    See number 6 and 7 in https://arxiv.org/pdf/1904.05760.pdf for details, and for empirical
+    comparison to other functions.
+    """
+
+    def __init__(
+        self,
+        batch_size: int,
+        num_objectives: int,
+        ideal_spec: Union[TensorType, IdealSpecCallable],
+        alpha: float = 0.0,
+    ):
+        """
+        :param batch_size: Number of elements in the batch, B.
+        :param num_objectives: Number of the objectives in the problem, K.
+        :param ideal_spec: This argument is used to determine how the ideal point is
+            calculated. If a Callable function specified, it is expected to take existing
+            ``models`` and ``datasets`` as arguments and return an ideal point with shape [K, 1]. 
+            If the Pareto front location is known, this arg can be used to specify a fixed ideal
+            point in each active learning iteration, as a tensor with minimum values, [K, 1].
+        :param alpha: Parameter for augmented Chebyshev, set to zero by default. A good
+            value for this parameter might be 0.0001, suggested in the reference above.
+        """
+        super().__init__(batch_size, num_objectives, ideal_spec)
+
+        tf.debugging.assert_non_negative(alpha, f"alpha should be nonnegative but received {alpha}")
+        self._alpha = alpha
+
+    def __repr__(self) -> str:
+        """"""
+        if callable(self._ideal_spec):
+            return (
+                f"Chebyshev({self._batch_size!r},"
+                f"{self._num_objectives!r},"
+                f"{self._ideal_spec.__name__},"
```

Comment:
You could easily remove the code duplication if you want:

```suggestion
                + (f"{self._ideal_spec.__name__}," if callable(self._ideal_spec) else f"{self._ideal_spec!r},") +
```